src/Browscap/Data/Useragent.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * This file is part of the browscap package.
+ *
+ * Copyright (c) 1998-2017, Browser Capabilities Project
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+namespace Browscap\Data;
+
+/**
+ * Class Useragent
+ *
+ * @category   Browscap
+ *
+ * @author     Thomas Müller <mimmi20@live.de>
+ */
+class Useragent
```

Comment:
`UserAgent`

Reply:
Also `final` as this is a VO

Reply:
If I make this class `final` its not possible to mock it for the tests.